pom.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.exist-db</groupId>
+        <artifactId>exist-apps-parent</artifactId>
+        <version>1.8</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>org.exist.xquery.modules</groupId>
+    <artifactId>jfreechart</artifactId>
+    <version>0.5.0-SNAPSHOT</version>
+
+    <name>JFreechart</name>
+    <description>JFreechart extension</description>
+    <url>https://github.com/eXist-db/jfreechart</url>
+
+    <organization>
+        <name>eXist-db</name>
+        <url>http://exist-db.org</url>
+    </organization>
+
+    <licenses>
+        <license>
+            <name>GNU Lesser General Public License, version 2.1</name>
+            <url>http://opensource.org/licenses/LGPL-2.1</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <url>https://github.com/eXist-db/jfreechart.git</url>
+        <connection>scm:git:https://github.com/eXist-db/jfreechart.git</connection>
+        <developerConnection>scm:git:https://github.com/eXist-db/jfreechart.git</developerConnection>
+    </scm>
+
+    <issueManagement>
+        <system>GitHub</system>
+        <url>https://github.com/eXist-db/jfreechart/issues</url>
+    </issueManagement>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.build.source>1.8</project.build.source>
+        <project.build.target>1.8</project.build.target>
+
+        <exist.version>4.6.0</exist.version>
+        <batik.version>1.11</batik.version>
+        <jfreechart.version>1.0.19</jfreechart.version>
```

Comment:
I see from https://github.com/jfree/jfreechart that the current version is 1.5.0 - and this page lists API changes between the 1.0.x and 1.5.0.

Reply:
there are indeed 2 issues: (1) availability in repos and (2) newer version (though not available on sf.net!

I did not want to do all changes in one go: the update to 1.5 we can do later. one step at the time :-)